README.md

```diff
@@ -87,6 +87,49 @@ For further information about debugging, please refer to the following documenta
 
 - [Debug process section](https://www.serverless.com/plugins/serverless-offline#usage-with-webpack)
 
+## Environmental variables
+
+The following variables should be present in the `.env.<NODE_ENV>` file.
+- AWS_PROVIDER_PROFILE=default
+- AWS_PROVIDER_REGION=eu-west-1
+- AWS_SERVER_PORT=3009
+- AWS_EVENT_BUS_NAME=default
+- AWS_EVENT_BUS_SOURCE=cvs.atf.daily.schedule
```

Comment:
Should this value be exposed publically?

Reply:
I can't think why it would be an issue? It's the value used by eventbridge to send it through matching rules.

Reply:
Is there any way that a third party could use this to send messages onto the queue?

Reply:
Not unless we gave them permissions to. I've swapped it to a generic name to match the other variables in the read.me.